src/main/cli/cli.c

```diff
@@ -3788,6 +3843,9 @@ static void cliDefaults(char *cmdline)
     resetConfigs();
 
     if (saveConfigs) {
+#ifdef USE_CLI_BATCH
+        resetCommandBatch();
```

Comment:
This should probably be outside of the `if` - if the config is wiped with a `defaults nosave` we effectively get a clean slate, and there wont be the potential for previous failed commands affecting us.

Reply:
Yes, that makes sense - I'll change.

Reply:
The downside is that it makes the ordering of the `batch start` command more sensitive. If it's not after the `defaults nosave` it won't catch any errors. But it is correctly positioned in the `diff` and `dump` outputs.

Reply:
I guess we could have the `defaults` reset just any errors that were found, but not change the on/off status for the batch mode.

Reply:
Yes, this will work. But it still kind of defeats the underlying goal but I'm not sure there's a better fix. The idea was to tell them of any errors in the block they pasted that might have scrolled off or otherwise obscured by the reboot. So logically commands that fail before the `defaults nosave` while not having any real effect, are technically still errors in the config they tried to use. But their config was wrong to begin with anyway and would never have worked so I'm thinking we can only go so far.

Reply:
Pushed a commit to only reset the error state on `defaults nosave`.